authenticate-users/composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "require": {
```

Comment:
Could we please also require PHP 7? `"php": ">=7"`

Reply:
yep!

Reply:
This actually needs to be 7.1 (for the void return type). Sorry about that!

Reply:
Friendly bump on this one

Reply:
oh yeah this was totally buried :P